Fixes redundancy in the Transforms migration section (#3013)
elastic/stack-docs: docs/en/install-upgrade/upgrading-stack.asciidoc

> If an index is less than 10GB of size, we recommend using {kib}'s Upgrade Assistant to automatically migrate the index.

Reviewer: ```suggestion
If an index size is less than 10 GB, we recommend using {kib}'s Upgrade Assistant to automatically migrate the index.
```

You can use the https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cat-indices-1[Get index information API] to obtain the size of an index:

[source,console]
----
GET _cat/indices/.transform-destination-example?v&h=index,store.size
----

> If an index size is greater than 10 GB, it is recommended to use the Reindex API. Reindexing consists of the following steps:

Reviewer: ```suggestion
If an index size is greater than 10 GB, we recommend using the Reindex API. Reindexing consists of the following steps:
```
